app/uk/gov/hmrc/customs/notification/services/WorkItemServiceImpl.scala

```diff
@@ -56,6 +62,14 @@ class WorkItemServiceImpl @Inject()(
     eventuallyProcessedOne
   }
 
+  lazy val registry: MetricRegistry = metrics.defaultRegistry
+
+  def recordTime(metric: String, workItem: WorkItem[NotificationWorkItem]): Unit = {
```

Comment:
strange name - there's no time recorded here. Maybe `recordCount` or `incrementCountMetric` or something like that?